apps/web/pages/[user].tsx

```diff
@@ -139,11 +189,18 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const ssr = await ssrInit(context);
   const crypto = require("crypto");
 
-  const username = (context.query.user as string).toLowerCase();
+  const usernameList = (context.query.user as string)
+    .toLowerCase()
+    .split("+")
+    .filter((el) => {
+      return el.length != 0;
+    });
```

Comment:
Parse the user query to identify number and slugs of users for dynamic group booking and make sure none of them is empty (if + is added at the end of the slug)

Reply:
Can this be move to a named function so it may be reutilised somewhere else. Also try to get a descriptive name to what it does.
Also != is required instead of !== 0?

Reply:
Id rather use `el.length > 0`